src/common/webpack/worker/web-worker.mts

```diff
@@ -1,6 +1,9 @@
 // eslint-disable-next-line camelcase
 declare let __webpack_public_path__: string;
 
+// @ts-expect-error ts does not find types in @okikio/sharedworker/@types/index.d.ts
```

Comment:
what type will be in project that will import SharedWorker form here? `unknown`?

Reply:
I checked, somehow types comes from `src/index.ts` inside package